pkg/kubelet/allocation/allocation_manager.go

```diff
@@ -214,6 +232,37 @@ func (m *manager) Run(ctx context.Context) {
 	}()
 }
 
+// Gernerate pod resize completed event
+func (m *manager) emitPodResizeCompletionEvent(resizedPod *v1.Pod) string {
```

Comment:
nit: make it clear that the resources in the pod spec are the allocated values

```suggestion
func (m *manager) emitPodResizeCompletionEvent(allocatedPod *v1.Pod) string {
```